sdk/python/kfp/cli/run.py

```diff
@@ -222,7 +222,7 @@ def _print_runs(runs: List[kfp_server_api.ApiRun], output_format: OutputFormat):
         run.created_at.isoformat(),
         next(rr
              for rr in run.resource_references
-             if rr.key.type == kfp_server_api.ApiResourceType.EXPERIMENT).id
+             if rr.key.type == kfp_server_api.ApiResourceType.EXPERIMENT).key.id
```

Comment:
Hey @Bobgy thanks for cleaning this up and apologies about the error this was a last minute change to make it a little nicer. Likely I've made the same mistake [here](https://github.com/zillow/pipelines/blob/3741cc8933b3ec8ae10480c37f28614a967cda3d/sdk/python/kfp/cli/pipeline.py#L217) and [here](https://github.com/zillow/pipelines/blob/3741cc8933b3ec8ae10480c37f28614a967cda3d/sdk/python/kfp/cli/pipeline.py#L255) too, do you mind fixing those quickly too please?

Reply:
Thanks for letting me know. Happy to fix them as well

Reply:
Fixed them too!